notes-json-kore.org

```diff
@@ -0,0 +1,96 @@
+#+Title: JSON format to represent Kore terms
```

Comment:
It's great for the reviewers to see the design process detailed out! But I'm not sure if we want to keep this file forever, I'd probably clean the info up a little so that it's clearer to users/other devs. Maybe it could be extracted into a some `docs/json.md` document, and it could also provide a link to the JSON-schema.